server/src/main/java/org/apache/seata/server/ServerApplication.java

```diff
@@ -18,15 +18,32 @@
 
 import java.io.IOException;
 
+import org.apache.seata.core.protocol.Version;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 /**
+ * Server Application
  */
 @SpringBootApplication(scanBasePackages = {"org.apache.seata"})
 public class ServerApplication {
     public static void main(String[] args) throws IOException {
         // run the spring-boot application
-        SpringApplication.run(ServerApplication.class, args);
+        SpringApplication app = new SpringApplication(ServerApplication.class);
+        // set the banner and version information
+        app.setBanner((environment, sourceClass, out) -> {
+            out.println("====================================================================");
+            out.println("                           _             _____            _        \n" +
```

Comment:
how about read from file ?

Reply:
Version information cannot be dynamically obtained from reading from a banner.txt file，i will try another way.
let me see see.

Reply:
I don't think it is a good choice to put this graph into the code, which will be less maintainable. Step back, you can read this graph and put it into the output stream.